src/components/Spinner/index.tsx

```diff
@@ -0,0 +1,58 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { Enum } from 'typescript-string-enums';
+import { Spinner as FabricSpinner, SpinnerSize as FabricSpinnerSize } from 'office-ui-fabric-react/lib/Spinner';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { SpinnerSize } from '../../util/enums/spinner';
+import './spinner.css';
+
+export { SpinnerSize };
+
+const SizeMap = {
+  [SpinnerSize.LARGE]: FabricSpinnerSize.large,
+  [SpinnerSize.MEDIUM]: FabricSpinnerSize.medium,
+  [SpinnerSize.SMALL]: FabricSpinnerSize.small,
+  [SpinnerSize.XSMALL]: FabricSpinnerSize.xSmall,
+};
+
+export interface SpinnerProps extends BaseComponentProps {
+  /**
+   * Label displayed below the spinner.
+   */
+  label?: string;
+  /**
+   * LARGE: 48px, MEDIUM: 40px, SMALL: 32px, XSMALL: 24px. Defaults to medium.
+   */
+  size?: SpinnerSize;
```

Comment:
Waiting for Design to come back with all scenarios we need to support.

After that, I'll consult with @BrendanMcK, see how we want to deal with labels and ARIA.